src/EA.Weee.RequestHandlers/Admin/AatfReports/GetAatfObligatedDataHandler.cs

```diff
@@ -65,9 +65,12 @@ public async Task<CSVFileData> HandleAsync(GetAatfObligatedData request)
                 }
             }
 
-            var fileName = string.Format("{0}-{1}{2}-{3:ddMMyyyy_HHmm}.csv",
-                 aatf.Name, request.ComplianceYear, (QuarterType)request.Quarter,
-                 DateTime.UtcNow);
+            var fileName = string.Format("{1}_{2}_{4}_Full data_{3:ddMMyyyy}_{3:HHmm}.csv",
+                 aatf.Name, 
+                 request.ComplianceYear, 
```

Comment:
Not related this change, but can we change this to use SystemTime.UtcNow and then update the unit test to ensure that the date time part is correct...